DashAISoftware/DashAI
# Change splits in Experiment configuration

# Summary


Adds back-end logic to support the change of splits configuration in Prepare Dataset Step.

## Type of change



- Front end new feature.
- Back end new feature.
- Refactoring.

## Changes


- Shows default splits in Prepare Dataset Step
- Save split configuration adding status to know is the splits have changed and if the new configuration is radom splits
- Use this new configuration in ModelJob flow by calling a new function update_dataset_splits
- Include tests for update_dataset_splits

## How to Test


1.- To check test run ```pytest tests/```
2.- To check correct functionality of this new feature from the front:
- Run ```python -m DashAI```
- Go to ```dashai/front``` and run ```yarn start```
- Create new experiments with different configuration of splits and verify the diffence between the results.
